### PR TITLE
#438: Better spacing for javadoc code examples

### PR DESCRIPTION
**Pull request by PascalSchumacher**
I tried to follow https://github.com/joel-costigliola/assertj-guava/commit/841ee1b60df12a59b5e7528901a07e9d18c0f618 

Once we are satisfied with this change I will create another pull request for the 3.0 classes.

### CODE REVIEW

**Comment by joel-costigliola** [comment]
Thanks a lot ! 
Don't create a PR for 3.x, 3.x is really just a branch containing java 8 specific stuff, I usually report issues in 2.x to 3.x by merging master (= latest 2.x) in java-8 branch (latest 3.x).

**Comment by PascalSchumacher** [comment]
Thanks for the info. :)

I will let you merge to the java-8 branch, but I guess there could be some java 8 specific classes e.g. AbstractLocalDateAssert which might need similar changes?

**Comment by joel-costigliola** [comment]
You are correct indeed, e.g. `AbstractOptionalAssert`, if you want to address it, just create another PR specific to those classes.

File: src/main/java/org/assertj/core/api/AbstractAssert.java [hunk 1]
```diff
@@ -83,9 +83,7 @@ protected WritableAssertionInfo getWritableAssertionInfo() {
    * defined by the user with {@link #overridingErrorMessage(String, Object...)}.
    * <p>
    * Example :
-   * 
-   * <pre><code class='java'>
-   * public TolkienCharacterAssert hasName(String name) {
+   * <pre><code class='java'> public TolkienCharacterAssert hasName(String name) {
```
Comment (PascalSchumacher) [inline src/main/java/org/assertj/core/api/AbstractAssert.java]: I'm wondering if the space between `<pre><code class='java'>` and the start of the code should be removed?

Comment (joel-costigliola) [inline src/main/java/org/assertj/core/api/AbstractAssert.java]: No because the next lines start also with a space, they should have the same indentation.

**Comment by joel-costigliola** [comment]
Integrated, many thanks !

**Pull request closed without merging**